### PR TITLE
Add SCD30 Component

### PR DESCRIPTION
**Pull request by brentru**
Adds support for the SCD-30 - NDIR CO2, Temperature, and Humidity Sensor (https://www.adafruit.com/product/4867)

### CODE REVIEW

File: components/i2c/scd30/definition.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "sensorType": "ambient-temp",
+      "defaultPeriod": 900
+    },
+    {
+      "displayName": "Humidity Sensor",
+      "sensorType": "humidity",
+      "defaultPeriod": 900
+    },
+    {
+        "displayName": "eCO2 Sensor",
```
Comment (ladyada) [inline components/i2c/scd30/definition.json:16]: not eCO2 - its true CO2!

**Review by ladyada — CHANGES_REQUESTED** [review]

File: components/i2c/scd30/definition.json (hunk 1 above)
Comment (brentru) [reply components/i2c/scd30/definition.json]: Ok - resolved in https://github.com/adafruit/Wippersnapper_Components/pull/13/commits/598eb64dd4bed797df1b602f7dbf97d846fd0f34

File: components/i2c/scd30/definition.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    {
+      "displayName": "Humidity Sensor",
+      "sensorType": "humidity",
+      "defaultPeriod": 900
+    },
+    {
+        "displayName": "CO2 Sensor",
+        "sensorType": "co2",
+        "defaultPeriod": 900
+      }
```
Comment (lorennorman) [inline components/i2c/scd30/definition.json:19]: superfluous indentation here

**Review by lorennorman — APPROVED** [review]
looks good!

File: components/i2c/scd30/definition.json (hunk 2 above)
Comment (brentru) [reply components/i2c/scd30/definition.json]: Fixed indentation in https://github.com/adafruit/Wippersnapper_Components/pull/13/commits/e05a526c13c8f20271a957fc55746069776bfefc

**Pull request merged**